packages/boot/test/bootstrapTests/price-feed-replace.test.ts

```diff
@@ -0,0 +1,200 @@
+/**
+ * @file  The goal of this test is  to see that the
+ * upgrade scripts re-wire all the contracts so new auctions and
+ * price feeds are connected to vaults correctly.
+ *
+ * 1. enter a bid
+ * 2. force prices to drop so a vault liquidates
+ * 3. verify that the bidder gets the liquidated assets.
+ */
+import { test as anyTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+
+import type { TestFn } from 'ava';
+import { ScheduleNotification } from '@agoric/inter-protocol/src/auction/scheduler.js';
+import { NonNullish } from '@agoric/internal';
+import { eventLoopIteration } from '@agoric/internal/src/testing-utils.js';
+import { Fail } from '@endo/errors';
+
+import {
+  LiquidationTestContext,
+  likePayouts,
+  makeLiquidationTestContext,
+  scale6,
+  LiquidationSetup,
+} from '../../tools/liquidation.js';
+import {
+  updateVaultDirectorParams,
+  updateVaultManagerParams,
+} from '../tools/changeVaultParams';
```

Comment:
missing a `.js`

This is typescript, so I guess it works for now, but I gather that's kind of a bug.